smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/XMLHttpResponseTraitWithHttpPayload.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.swift.codegen.integration.httpResponse.bindingTraits
+
+import software.amazon.smithy.codegen.core.CodegenException
+import software.amazon.smithy.model.shapes.Shape
+import software.amazon.smithy.model.shapes.ShapeType
+import software.amazon.smithy.model.traits.StreamingTrait
+import software.amazon.smithy.swift.codegen.ClientRuntimeTypes
+import software.amazon.smithy.swift.codegen.SwiftTypes
+import software.amazon.smithy.swift.codegen.SwiftWriter
+import software.amazon.smithy.swift.codegen.declareSection
+import software.amazon.smithy.swift.codegen.integration.HttpBindingDescriptor
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+import software.amazon.smithy.swift.codegen.integration.httpResponse.HttpResponseBindingRenderable
+import software.amazon.smithy.swift.codegen.integration.serde.xml.MemberShapeDecodeXMLGenerator
+import software.amazon.smithy.swift.codegen.model.hasTrait
+import software.amazon.smithy.swift.codegen.model.isEnum
+
+class XMLHttpResponseTraitWithHttpPayload(
+    val ctx: ProtocolGenerator.GenerationContext,
+    val binding: HttpBindingDescriptor,
+    val writer: SwiftWriter,
+    val shapeContainingMembers: Shape,
+) : HttpResponseBindingRenderable {
+
+    override fun render() {
```

Comment:
Renders statements to set output model values from HTTP payload.
Used to construct a HTTP binding for an output model.